build/Makefile.kits

```diff
@@ -5,12 +5,15 @@
 # 2007-08-23
 #--------------------------------------------------------------------------------
 
+OUTPUT_PACKAGE_PREFIX=
+
 ifneq ($(COMBINED_PACKAGES),1)
   DATAFILES = Base_SCXCore.data $(PF_DEPENDENT_DATAFILES)
   ifeq ($(PF_ARCH),ppc)
     ifeq ($(PF_DISTRO),REDHAT)
       SCX_PREFIX=scx
       OUTPUT_PACKAGE_PREFIX=$(SCX_PREFIX)-$(SCX_BUILDVERSION_MAJOR).$(SCX_BUILDVERSION_MINOR).$(SCX_BUILDVERSION_PATCH)-$(SCX_BUILDVERSION_BUILDNR).rhel.$(PF_MAJOR).$(PF_ARCH)
+      OUTPUT_PACKAGE_SPECIFICATION=--OUTPUTFILE=$(OUTPUT_PACKAGE_PREFIX)
```

Comment:
Maybe also initialize OUTPUT_PACKAGE_SPECIFICATION otherwise in the installbuilder part, it uses the side effect that an undefined variable in bash resolves to an empty string.